data/core/statusview.lua

```diff
@@ -124,7 +124,7 @@ function StatusView:get_items()
       col > config.line_limit and style.accent or style.text, "col: ", col,
       style.text,
       self.separator,
-      string.format("%d%%", line / #dv.doc.lines * 100),
+      string.format("%d%%", math.floor(line / #dv.doc.lines * 100)),
```

Comment:
a change in some lua version made it so that divisions on integers turn them into floats, which is not the input thats expected here

Reply:
Yes but that would change the indicator for how far you are into the file

Reply:
Fixed in forced push